content/docs/reference/downstream-mtls-settings.mdx

```diff
@@ -3,6 +3,7 @@
 
 id: downstream-mtls-settings
 title: Downstream mTLS Settings
+description: Add mTLS settings and manage client certificate requirements for end users connecting to Pomerium-managed routes.
```

Comment:
nit: this could probably be just "Manage client certificate requirements for end users connecting to Pomerium-managed routes."

Reply:
It could, but including "mTLS" in the description is important because the description acts as the meta description, which appears on the SERPs. We can refine this, but it's mainly there to for that reason.

Reply:
These descriptions are tricky because they are sometimes pulled and served by Google as the meta description on the SERPs, and they also appear in our docs when you search for a setting. Because this description speaks for the whole page, I wanted to include "mTLS" as a keyword. It could be revised to less clunky and more fluid, though.

Reply:
OK, here's another suggestion (I'll leave it up to you):

"Downstream mTLS settings configure the client certificate requirements for end users connecting to Pomerium-managed routes."

Reply:
@kenjenkins thanks for workshopping that. I'll update it and merge!